tests/test_special_euclidean.py

```diff
@@ -93,6 +93,12 @@ def setUp(self):
                 'point_1': point_1,
                 'point_2': point_2}
 
+        else:  # matrix_from_vector fails with tensorflow
```

Comment:
Remove this omment

Reply:
Is there a more "official" way to pinpoint things that we know need to be improved or fixed ?

Reply:
Yes!
- TODO(pchauchat) with an actionable item,
- FIXME
- issues, that can refer lines in the code, or even the corresponding TODOs.

Having a standardized syntax for TODOs and FIXMEs allows to use numerous online tools, for example:
https://github.com/naholyr/github-todos